packages/vscode-extension/src/handlers.ts

```diff
@@ -424,6 +425,16 @@ function registerCoreEvents() {
   }
 }
 
+export async function getAzureProjectConfigV3(): Promise<ProjectConfigV3 | undefined> {
```

Comment:
Is the legacy getAzureSolutionSettings() function still used in other places? If not, why not just replace with v3 logic?

Reply:
this is a workaround, once v3 release, the projectSettings.json file may not be exist, so later will udpate the logic.